src/routes/blogs.rs

```diff
@@ -90,6 +155,12 @@ pub fn create(form: LenientForm<NewBlogForm>, rockets: PlumeRocket) -> RespondOr
     let intl = &rockets.intl.catalog;
     let user = rockets.user.clone().unwrap();
 
+    let custom_domain = if *(&form.custom_domain.is_empty()) {
```

Comment:
clippy says 
```suggestion
    let custom_domain = if form.custom_domain.is_empty() {
```

Reply:
I can't accept your proposal via Github (apparently I don't have the right to push to this repository :upside_down_face:), but I made it manually, thanks.